src/platform_impl/windows/window.rs

```diff
@@ -698,6 +698,14 @@ impl Window {
         Some(self.window_state_lock().current_theme)
     }
 
+    #[inline]
+    pub fn title(&self) -> String {
+        let len = unsafe { GetWindowTextLengthW(self.window.0) };
+        let mut buf = vec![0; (len + 1) as usize];
+        unsafe { GetWindowTextW(self.window.0, buf.as_mut_ptr(), len) };
+        String::from_utf16_lossy(&buf[..len as _])
```

Comment:
Let's use the `util::decode_wide` function.
I tested it on the `window` example and it dropped the `!` at the end and includes a `\0`.
Below an implementation suggestion which returns the expected string.
```rust
let len = unsafe { GetWindowTextLengthW(self.window.0) } + 1; // incl. NULL terminator
let mut buf = vec![0; len as usize];
unsafe { GetWindowTextW(self.window.0, buf.as_mut_ptr(), len) };
util::decode_wide(&buf).to_string_lossy().to_string()
```

Reply:
Doesn't that result in an extra allocation?

Reply:
fine for me - it's used in other places already and this function shouldn't be called a lot imo

Reply:
done